### DISCUSSION TITLE
can Spacy compute collocations?

### DISCUSSION THREAD

**Discussion by randomgambit** [explosion/spaCy#10926] [Help: Coding & Implementations]
I know that collocations (statistical tests of associations between words in a text) can be computed with `nltk`. However, I was wondering if the same could be done with `spacy`? Perhaps with better speed?

Thanks

**Comment by polm** [comment]
I am not familiar with the NLTK feature, but looking at the docs [here](https://www.nltk.org/howto/collocations.html) it looks like it's a feature for extracting n-grams, possibly with filtering. You can do that with spaCy but there's no built-in feature for it.
